hw/sensor/include/sensor/sensor.h

```diff
@@ -326,6 +336,7 @@ typedef int (*sensor_set_trigger_thresh_t)(struct sensor *, sensor_type_t,
 struct sensor_driver {
     sensor_read_func_t sd_read;
     sensor_get_config_func_t sd_get_config;
+    sensor_config_func_t sd_config;
```

Comment:
Since we have sensor_get_config_func_t I suggest to name the new one sensor_set_config_func_t